do-catalog/src/components/catalogSearch/FilterDetail.vue

```diff
@@ -23,7 +23,7 @@ export default {
   },
   computed: {
     ...mapState({
-      filterMetadata: function(state) { return state.doCatalog.filtersAvailable[this.filterId]; }
+      filterMetadata: state => state.doCatalog.filtersAvailable[this.filterId]
```

Comment:
If you use arrow functions, `this` will be undefined.
By the way, using
```js
varName1: function(param) { ... }
varName2: param => ...
```
is just to unify with the rest of computed vars and make it easier to read.

Reply:
OK, I saw other parts of the code using arrow functions for computed props. I would try to use the same style in all the code.